app/controllers/proposals_controller.rb

```diff
@@ -61,7 +61,12 @@ def complete
   def query
     check_search_params
     query_listing = listing
-    @proposals_data = query_listing.query
+    begin
```

Comment:
Should we have this in `#download` as well (and possibly the report generation) or is that overkill?